ext/mbstring/php_unicode.c

```diff
@@ -177,7 +176,7 @@ static unsigned php_unicode_totitle_raw(unsigned code, enum mbfl_no_encoding enc
 	return php_unicode_toupper_raw(code, enc);
 }
 
-unsigned php_unicode_tofold_raw(unsigned code, enum mbfl_no_encoding enc)
+static unsigned php_unicode_tofold_raw(unsigned code, enum mbfl_no_encoding enc)
```

Comment:
No idea why this function was not `static` before. I can't find any callers outside this file. Guessing that it was a mistake.

Reply:
Yes, that looks like an oversight.

Reply:
Supporting the point... `php_unicode_tofold_raw` wasn't declared in `php_unicode.h`.